tools/haddock.sh

```diff
@@ -15,4 +16,5 @@ mkdir -p $DOCS
 cp $DOC_ROOT/{*.html,*js,*.png,*.gif,*.css} $DOCS
 cp -a $DOC_ROOT/tensorflow* $DOCS
 rm -f $DOCS/*/*.haddock
+ln -s $DOCS/tensorflow-$RECENT_RELEASE/TensorFlow-Core.html $DOCS/tensorflow/TensorFlow-Core.html
```

Comment:
How about something more direct (as there's only a single directory):
ln -s $DOCS/tensorflow-* $DOCS/tensorflow

RECENT_RELEASE then become unnecessary.

Reply:
Awesome! I like that a lot more! I updated. Thank you for taking the time to show me that!